### ISSUE TITLE
Problem with the hiding columns

### ISSUE THREAD

**Issue by Miyuki0** [jacekkopecky/living-meta-analysis#63]
When in localstorage, problems with hidden columns.

```
Uncaught TypeError: Cannot read property 'dataset' of null
    at HTMLDivElement.unhideColumns (metaanalyses.js:2970)
```

**Comment by jacekkopecky** [comment]
only happens on the last column because the last table column doesn't have fullcolinfo popupbox

**Comment by jacekkopecky** [comment]
fixed in #70
